agent/src/main/java/com/teamscale/jacoco/agent/commit_resolution/git_properties/GitMultiProjectPropertiesLocator.java

```diff
@@ -72,7 +72,7 @@ void searchFile(File file, boolean isJarFile) {
 					logger.debug(
 							"Found inconsistent git.properties file: the git.properties file in {} either does not specify the" +
 									" Teamscale project (teamscale.project) property, or does not specify the commit " +
-									"(git.commit.id, git.commit.branch + git.commit.time, or teamscale.timestamp) or ." +
+									"(git.commit.id, git.commit.branch + git.commit.time, or teamscale.commit.branch + teamscale.commit.time) or ." +
```

Comment:
Should we use the constants from `GitPropertiesLocatorUtils` to have them defined in one place?